appendix-basic65.tex

```diff
@@ -672,17 +776,18 @@ \subsection{ASC}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$C6
 \item [Format:] {\bf ASC(}string{\bf)}
-\item [Usage:] Takes the first character of
-               the string argument and returns its numeric code value.
-               The name was apparently chosen to be a mnemonic to ASCII,
-               but the returned value is in fact the so-called PETSCII code.
+\item [Usage:] The PETSCII code of the first character of
+               the string argument, as a number.
 \item [Remarks:]
                {\bf ASC} returns zero for an empty string, whose behaviour
                is different to BASIC 2, where \screentext{ASC("")} gave an error.
```

Comment:
> whose behaviour is different to BASIC 2

This was existing text, but still maybe it could be a chance to change it to something like:
- "a behaviour that differs to BASIC 2,"

Reply:
I gather that "different to" is acceptable in British English, but "different from" is more universally common. I have changed all instances.

Reply:
The team made the conscious decision to go with Australian English for the user guide over my preference for American English. As such, "different to" is the correct usage.

Reply:
Thanks for mentioning it. I'll change it back to "different to." It's not a big enough difference to override a previous style decision.

Reply:
Oh, reflecting back on this, I think the aspect that was bugging me was more the use of the word "whose" (wasn't as touchy about "different from" vs "different to"), so my suggested re-wording was more an attempt to avoid using "whose", which sounds a little odd to my ears. "Whose" feels more natural to my ears for living things ("Mary had a little lamb, whose fleece was white as snow"), but feels out of place here.

Ok, googling now, I see examples of "whose" being used for inanimate objects too, though the examples all seem to from classy literature :D (Sure, our mega65 manual is classy in it's own way too, ofcourse ;) )

Reply:
(I'll just mark this one as unresolved for now, just to assure to hear your thoughts on it)

Reply:
I fixed it: https://github.com/MEGA65/mega65-user-guide/commit/5c66d405c9ee8ab622817a0d9681f3044fe25fd8